web/src/routes/execution.tsx

```diff
@@ -71,9 +74,75 @@ export function ExecutionRoute() {
       {execution ? (
         <div>
           <h2 id="execution-name-header" className="align-self-center mt-3">
-            {execution.name}-{executionId}
+            {execution.name} - {executionId}
           </h2>
           <ExecutionStatus execution={execution} />
+          <section id="notification-input" className="mt-3">
+            <CsrfForm id="notification-input-form" method="POST" className="d-flex flex-column mb-3">
+              <div className="d-flex mt-2">
+                <FormLabel className="fs-5 mt-2 me-4">Nachrichten</FormLabel>
+                <div className="d-flex justify-content-center ms-auto">
+                  <Button className="align-self-center" type="submit">
```

Comment:
I would suggest to disable the button as long as the execution was not started yet.